kfs-core/src/main/java/edu/arizona/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -1,27 +1,205 @@
 package edu.arizona.kfs.fp.document;
 
 import org.apache.log4j.Logger;
+import org.kuali.kfs.fp.document.DisbursementVoucherConstants;
+import org.kuali.kfs.sys.businessobject.Bank;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntry;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntrySequenceHelper;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntrySourceDetail;
 import org.kuali.kfs.sys.businessobject.SourceAccountingLine;
-
 import edu.arizona.kfs.fp.businessobject.DisbursementVoucherSourceAccountingLineExtension;
+import edu.arizona.kfs.fp.businessobject.PaymentMethod;
+import edu.arizona.kfs.fp.service.PaymentMethodGeneralLedgerPendingEntryService;
+import edu.arizona.kfs.sys.KFSConstants;
+import edu.arizona.kfs.sys.KFSPropertyConstants;
+import edu.arizona.kfs.vnd.businessobject.VendorDetailExtension;
+import org.kuali.rice.krad.service.BusinessObjectService;
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.vnd.businessobject.VendorAddress;
+import org.kuali.kfs.vnd.businessobject.VendorDetail;
+import org.kuali.rice.kns.service.DocumentHelperService;

```

Comment:
org.kuali.rice.kns.service.DocumentHelperService has been deprecated. Should use org.kuali.rice.krad.service.DocumentDictionaryService.